app/(pages)/things-every-employee-should-know/crypto-wallets/page.tsx

```diff
@@ -0,0 +1,32 @@
+"use client";
+import { motion } from "framer-motion";
+import { Typography } from "@/components/ui/typography";
+import Link from "next/link";
+
+export default function GitBasics() {
+    return (
+        <motion.div
+            initial={{ opacity: 0, y: 20 }}
+            animate={{ opacity: 1, y: 0 }}
+            exit={{ opacity: 0, y: -20 }}
+            transition={{ duration: 0.5 }}
+            className="container mx-auto px-4 py-8"
+        >
+            <Typography variant="h1" className="mb-8 text-4xl border-b border-red-500 pb-2">
+                Crypto Wallets
+            </Typography>
+            <Typography variant="body" className="mb-4">
+                There are lots of options when it comes to what type of wallet you can use. For the sake of simplicity, I’m going to suggest you get started with Bitcoin core (https://bitcoin.org/en/download). This is an application that you install and will set up a wallet for you. Two things to note: 1. This will take up A LOT of space (around 40gigs at the time that I write this) since it syncs the whole blockchain to your hard drive and 2. Your wallet will be a file on your hard drive so if your hard drive dies your Bitcoin is all lost.
+
```

Comment:
Newlines and whitespace are not respected by HTML tags. Use separate components or a break tag <br/> for spacing.